_posts/2022-07-19-react-to-full-stack.md

```diff
@@ -0,0 +1,606 @@
+---
+layout: post
+title: React To Full Stack
+date: 2022-07-19
+tags: [react, mui, typescript, aws]
+author: Dale Murugan
+image: /img/fivetranslacklambdalogo.png
+description: "Level-up from a React to a Full Stack developer. A definitive guide for junior developers"
+---
+
+![React To Full Stack](/img/react-to-full-stack/react-to-full-stack-main.png)
+
+<br/>
+
+React seems to be the starting place for a lot of new developers, but what do you learn next? I know from personal experience that it can be daunting as a junior developer trying to decide what backend tech stack to go with. Should you use a SQL or NoSQL database? Should you use REST or GraphQL? How do you manage authentication? Should you learn .Net or Node? What should you use for hosting? What even is a Docker?
+
+This article will save you the decision fatigue. Assuming that you know a bit of React, you can follow this step-by-step guide to build and deploy a full stack cloud native application, using AWS's serverless infrastructure.
+
+---
+
+<br>
+## Overview
+ We are going to create the backend for a todo react app that I prepared earlier. The app is built with typescript and styled with material UI (MUI); we won't be covering either of these as there are plenty of useful free resources on these topics online. Instead, we will be focusing on creating the backend infrasture and hosting the application.
```

Comment:
I didn't see any `TS ` examples below, they all look like raw `JS`, which is fine, just needs to be consistent.

Reply:
Yeah the `TS` is referring to the actual todo-app repo, not the code snippets in the blog post